py/jupyterlite/src/jupyterlite/app.py

```diff
@@ -82,9 +80,6 @@ class BaseLiteApp(JupyterApp, LiteBuildConfig, DescribedMixin):
             # server-specific things
             "port": "LiteBuildConfig.port",
             "base-url": "LiteBuildConfig.base_url",
-            # pyolite things likely to move away
-            "piplite-wheels": "LiteBuildConfig.piplite_urls",
```

Comment:
so this is big-break town. I think we need to solve this issue, potentially on a separate PR first. I'll start an issue...

Reply:
Created https://github.com/jupyterlite/jupyterlite/issues/929